src/basic/def.h

```diff
@@ -3,75 +3,70 @@
 
 #include "util.h"
 
-#define DEFAULT_TIMEOUT_USEC (90*USEC_PER_SEC)
-#define DEFAULT_RESTART_USEC (100*USEC_PER_MSEC)
-#define DEFAULT_CONFIRM_USEC (30*USEC_PER_SEC)
+#define DEFAULT_TIMEOUT_USEC (90 * USEC_PER_SEC)
+#define DEFAULT_RESTART_USEC (100 * USEC_PER_MSEC)
+#define DEFAULT_CONFIRM_USEC (30 * USEC_PER_SEC)
 
-#define DEFAULT_START_LIMIT_INTERVAL (10*USEC_PER_SEC)
+#define DEFAULT_START_LIMIT_INTERVAL (10 * USEC_PER_SEC)
 #define DEFAULT_START_LIMIT_BURST 5
 
 /* The default time after which exit-on-idle services exit. This
  * should be kept lower than the watchdog timeout, because otherwise
  * the watchdog pings will keep the loop busy. */
-#define DEFAULT_EXIT_USEC (30*USEC_PER_SEC)
+#define DEFAULT_EXIT_USEC (30 * USEC_PER_SEC)
 
 /* The default value for the net.unix.max_dgram_qlen sysctl */
 #define DEFAULT_UNIX_MAX_DGRAM_QLEN 512UL
 
-#define SIGNALS_CRASH_HANDLER SIGSEGV,SIGILL,SIGFPE,SIGBUS,SIGQUIT,SIGABRT
+#define SIGNALS_CRASH_HANDLER SIGSEGV, SIGILL, SIGFPE, SIGBUS, SIGQUIT, SIGABRT
 #define SIGNALS_IGNORE SIGPIPE
 
 #if HAVE_SPLIT_USR
-#define KBD_KEYMAP_DIRS                         \
-        "/usr/share/keymaps/\0"                 \
-        "/usr/share/kbd/keymaps/\0"             \
-        "/usr/lib/kbd/keymaps/\0"               \
+#define KBD_KEYMAP_DIRS             \
+        "/usr/share/keymaps/\0"     \
+        "/usr/share/kbd/keymaps/\0" \
+        "/usr/lib/kbd/keymaps/\0"   \
         "/lib/kbd/keymaps/\0"
 #else
-#define KBD_KEYMAP_DIRS                         \
-        "/usr/share/keymaps/\0"                 \
-        "/usr/share/kbd/keymaps/\0"             \
+#define KBD_KEYMAP_DIRS             \
+        "/usr/share/keymaps/\0"     \
+        "/usr/share/kbd/keymaps/\0" \
         "/usr/lib/kbd/keymaps/\0"
 #endif
 
-/* Note that we use the new /run prefix here (instead of /var/run) since we require them to be aliases and that way we
- * become independent of /var being mounted */
+/* Note that we use the new /run prefix here (instead of /var/run) since we require them to be aliases and
+ * that way we become independent of /var being mounted */
 #define DEFAULT_SYSTEM_BUS_ADDRESS "unix:path=/run/dbus/system_bus_socket"
 #define DEFAULT_USER_BUS_ADDRESS_FMT "unix:path=%s/bus"
 
-#define PLYMOUTH_SOCKET {                                       \
-                .un.sun_family = AF_UNIX,                       \
-                .un.sun_path = "\0/org/freedesktop/plymouthd",  \
-        }
+#define PLYMOUTH_SOCKET \
+        { .un.sun_family = AF_UNIX, .un.sun_path = "\0/org/freedesktop/plymouthd", }
```

Comment:
hmm, i preferred the old indentation here... structure initializations shouldn't be merged into one line, never